client/css/style.css

```diff
@@ -815,25 +815,30 @@ button {
 #chat .time,
 #chat .from,
 #chat .text {
-	display: table-cell;
+	display: block;
 	padding: 2px 0;
-	vertical-align: top;
+	flex: 0 0 auto;
 }
 
 #chat .time {
 	color: #ddd;
 	text-align: right;
-	max-width: 46px;
-	min-width: 46px;
+	width: 46px;
```

Comment:
I remember I had to set these 2 explicitly, but I can't remember why specifically: https://github.com/thelounge/lounge/commit/8118d56e16d2ab9a2321badffb4ace2d08d5c278#diff-97db1f70168fb5f12457b238ff6052b5L771

Chances are, this was table-layout-specific anyway...